admin/package.json

```
@@ -4,8 +4,8 @@
     "scripts": {
         "postinstall": "cd server && npm install",
         "serve": "cd server && npm run serve",
-        "start": "npm run intl:compile && run-p gql:types generate-block-types && dotenv -c secrets -- vite",
-        "build": "npm run intl:compile && run-p gql:types generate-block-types && NODE_ENV=production dotenv -c secrets -- vite build",
+        "start": "npm run intl:compile && run-p gql:types generate-block-types && dotenv -c secrets -e .env.site-configs -- vite",
+        "build": "npm run intl:compile && run-p gql:types generate-block-types && NODE_ENV=production dotenv -c secrets -e .env.site-configs -- vite build",
```

```suggestion
        "start": "npm run intl:compile && run-p gql:types generate-block-types && dotenv -c secrets -e site-configs -- vite",
        "build": "npm run intl:compile && run-p gql:types generate-block-types && NODE_ENV=production dotenv -c secrets -e site-configs -- vite build",
```

Shouldn't this be just `site-configs` (as for `secrets`)?

---

yes a bit confusing to mix -c and -e, maybe changing both to -e would be better